nbgrader/docs/source/user_guide/installation.rst

```diff
@@ -15,29 +15,34 @@ Or, if you use `Anaconda <https://www.continuum.io/downloads>`__::
 
 nbgrader extensions
 -------------------
-You may then install the nbgrader extensions for Jupyter notebook. This will
-install both the *create assignment* toolbar extension and *assignment list*
-notebook server extension::
 
-    nbgrader extension install
+You may then install the nbgrader extensions for Jupyter notebook. Previously
+this was done using the ``nbgrader extension install`` command. However, moving
+forward this is done using the ``jupyter nbextension`` and ``jupyter serverextension``
+commands.
 
-To use the toolbar extension as either an instructor or a student, activate the
-extension with::
+To install and enable the frontend nbextensions (*assignment list* and
+*create assignment*) run:
 
-    nbgrader extension activate
+    jupyter nbextension install --sys-prefix --py nbgrader
+    jupyter nbextension enable --sys-prefix --py nbgrader
+    
+To install the server extension for *assignment_list* run:
 
-If you want to install the extension for only your user environment and not
-systemwide, use ``nbgrader extension install --user``. If you don't want to
-have to reinstall the extension when nbgrader is updated, use ``nbgrader
-extension install --symlink``. If you want to only install a specific
-extension, use ``nbgrader extension install <name>``, where ``<name>`` is the
-name of the extension you want (e.g. ``create_assignment``).
+    jupyter serverextension enable --sys-prefix --py nbgrader
 
-To get help and see all the options you can pass while installing or activating
-the nbgrader notebook extension, use::
+In both the ``nbextension`` and ``serverextension`` commands:
 
-    nbgrader extension install --help-all
-    nbgrader extension activate --help-all
+* To install for all users, replace `--sys-prefix` by `--system`.
+* To install only for the current user replace `--sys-prefix` by `--user`.
```

Comment:
Yep, will clarify and expand this section as I know lots of folks deploying nbgrader want to do this stuff.